test/helper.js

```diff
@@ -0,0 +1,17 @@
+const mvm = require('mongodb-version-manager');
```

Comment:
Can we move all of this to the `.travis.yml` file, rather than requiring it every time we run the tests locally? The version manager installs a [command line utility](https://github.com/mongodb-js/version-manager/blob/master/bin/m.js), so you can call `m $MONGODB_VERSION`

Reply:
I don't think `.travis.yml` is the best place because otherwise a user can't locally do something like `MONGODB_VERSION=3.2.x npm test` since this logic is in `.travis.yml`. Rather, I think a more appropriate place is in `package.json` in `pretest`.

But, this was my original intent but I ran into the issue of setting a default value for `MONGODB_VERSION`. Here is what I was thinking: `"pretest": "cross-env MONGODB_VERSION=${MONGODB_VERSION:=4.x} m use $MONGODB_VERSION",`

But, Windows does not support the default value syntax and it is not supported in `cross-env` either. A fork was made by a user to implement this functionality but my concern was that it likely won't stay up to date with the `cross-env` counterpart.
[https://github.com/kentcdodds/cross-env/pull/157](https://github.com/kentcdodds/cross-env/pull/157)
https://www.npmjs.com/package/@naholyr/cross-env

Reply:
If we are going to introduce this directly into the test runner, then I think we should do a few things:
  - provide a way to bypass downloading anything. We shouldn't force developers to sit through a full download of the server each time they want to run. In the node driver's test runners we provide a command line argument `-s/--skip` to bypass this, here we might have to use a special value for `MONGODB_VERSION` 
  - document the introduced environment variables (probably in the `README.md`)

Reply:
@mbroadst See latest changes accommodating your requests.

I decided **against** switching from using `promisify` because it makes more sense now adding the skip functionality. Also, I didn't want to remove the `Finished installing ...` text because it is useful to inform the user that MongoDB is still installing and when it is finished.